orchestration-scripts/configure_docker_compose.py

```diff
@@ -141,6 +142,14 @@ def save_file_here(contents: str, name: str) -> None:
     file.write(contents)
     print(f"Docker-Compose File saved at {os.getcwd()}/{name}!")
 
+def copy_experiment_requirements(experiment_path: str):
```

Comment:
`configure_docker_compose.py` shouldn't be responsible for doing this, an action that modifies another part of the filesystem. While it's useful, it's an unexpected side-effect. Is there anywhere else we could easily move this action to ("easy" = 30 mins of work or less)? If not, add a note about this to the `Technical Debt` section at the bottom of the main README.

Reply:
I've made a note of this within the technical debt section, but the reason we have the code within this file is that Python is really finicky with imports and can't import from any folder with a dash in its name. So we can't import from `orchestration-scripts` and thus we can't move the code into a new file without renaming the entire folder, which means we would have to change the filepath location everywhere else it is used as well.